Don't build the game_test target.
It isn't set up properly for macOS and can't be linked.

https://github.com/raceintospace/raceintospace/issues/307#issuecomment-637153218
--- src/game/CMakeLists.txt.orig	2020-08-02 14:14:44.000000000 -0500
+++ src/game/CMakeLists.txt	2020-08-04 05:07:45.000000000 -0500
@@ -117,10 +117,3 @@
   ${test_dir}/game/roster_test.cpp
   )
 
-add_executable(game_test ../../test/test_main.cpp ${test_sources} ${game_sources})
-set_target_properties(game_test PROPERTIES COMPILE_FLAGS "-DBOOST_TEST_NO_LIB=1")
-target_link_libraries(game_test ${game_libraries})
-add_test(
-  NAME game_test
-  COMMAND game_test --catch_system_errors=yes
-  )
